src/components/logo/logo.js

```diff
@@ -20,31 +21,22 @@ const mapStateToProps = state => ({
 });
 
 @connect(mapStateToProps)
+@withPlayer
+@withLogger(COMPONENT_NAME)
 /**
  * Logo component
  *
  * @class Logo
- * @example <Logo player={this.player} />
- * @extends {BaseComponent}
+ * @example <Logo />
+ * @extends {Component}
  */
-class Logo extends BaseComponent {
+class Logo extends Component {
   /**
    * should render component
-   * @param {*} props - component props
    * @returns {boolean} - whether to render the component
-   * @static
-   */
-  static shouldRender(props: any): boolean {
-    const componentConfig = props.config.components[COMPONENT_NAME.toLocaleLowerCase()];
-    return !(Object.keys(componentConfig).length === 0 && componentConfig.constructor === Object);
-  }
-  /**
-   * Creates an instance of Logo.
-   * @param {Object} obj obj
-   * @memberof Logo
    */
-  constructor(obj: Object) {
-    super({name: COMPONENT_NAME, player: obj.player});
+  _shouldRender(): boolean {
```

Comment:
is it here on purpose?

Reply:
so the shouldRender was there, see a few rows above :-)